### ISSUE TITLE
Incorrect use of GlobalState 

### ISSUE THREAD

**Issue by deswif** [guilhermetog/flutter_view_controller#4]
GlobalState is available for the whole application, as GetIt. I dont think thats such a nice idea

**Comment by guilhermetog** [comment]
I agree! I implemented it in version 0.0.7 as a way to rebuild Widget Tree, over external events (like keyboard opening).
But I ended up to using it for another things and made it available for final usage.

My initial goal was to build something as powerful as GetIt, but smaller and simpler. So from version 0.1.21 I started to remove (or separated in another packages) things that shouldn't be part of FVC.

Today GlobalState is doing nothing for the main structure, and is very simple to be implemented, in case someone wants it, so it will be removed in the next version. (0.1.24)
